apps/tests/aws-runtime/test/test-service.ts

```diff
@@ -582,60 +586,62 @@ export const onEntityEvent = subscription(
   "onEntityEvent",
   { events: [entityEvent] },
   async ({ id }) => {
-    const value = await counter.get(id);
-    await counter.set(id, { n: (value?.n ?? 0) + 1 });
+    const value = await counter.get([id]);
```

Comment:
why doesn't `get` accept an object with the key properties? That would align with get/set and likely easier to target with AI since key-value pairs are independent of order

Reply:
It does (see PR description)

Reply:
The Tuple and Map formats are interchangeable.

Reply:
We can leave it for now :)